ramoun/chapter_02/ex2_2.cpp

```diff
@@ -0,0 +1,12 @@
+// ex2_2.cpp
+// genrate table
+#include <iostream>
+#include <iomanip>
+
+using namespace std;
+
+int main() {
```

Comment:
### better be consistent with coding style

- braces on the same level as code level
```cpp
int main () 
    {
    // code level
    }
```
- braces on their own level:
```cpp
int main() {
    // code level
}
```